crates/server-primitives/src/admin.rs

```diff
@@ -6,6 +6,8 @@ use serde_json::Value;
 pub struct InstallApplicationRequest {
     pub application: calimero_primitives::application::ApplicationId, // TODO: rename to application_id
     pub version: semver::Version,
+    pub path: String,  // represents the url path to the release binary (e.g. ipfs path)
```

Comment:
why not just use url here?

Reply:
i actually updated the name to use url so its more clear. path can also represent file dirs just to avoid confusion what this param is...

Reply:
Aha, in my PR I'm using source URIs. Will coalesce.